SKILL.tsv

```diff
@@ -4335,7 +4335,7 @@ SKILL_20150714_004334	$Throwing Spear: Bind
 SKILL_20150714_004335	$Enemies hit by [Throwing Spear] have a chance to be inflicted with the [Bind] status for 7 seconds. Increases the chance by 15% per attribute level. 
 SKILL_20150714_004336	$Increases the damage dealt on an enemy with [Embowel] by 1% per attribute level.
 SKILL_20150714_004337	$Increases the damage dealt on an enemy with [Stomping Kick] by 1% per attribute level.
-SKILL_20150714_004338	$Increases the amount of enemies losing defense from [Warcry] by 1 per attribute level. 
+SKILL_20150714_004338	$Increases the amount of targets that [Warcry] affects by 1 per attribute level. 
```

Comment:
Increases the amount of targets that [Warcry] affects by 1 per attribute level. 

to

Increases the amount of enemies losing their defense affected from [Warcry] by 1 per attribute level.